src/ibusenginesimple.c

```diff
@@ -428,6 +428,60 @@ ibus_engine_simple_commit_str (IBusEngineSimple *simple,
     g_free (backup_str);
 }
 
+static gboolean
+check_compose_sequence (guint     *keys,
+                        int        n_keys,
+                        GString   *output)
```

Comment:
It might be copied my mistake.

```suggestion
check_compose_sequence (guint   *keys,
                        int      n_keys,
                        GString *output)
```